### ISSUE TITLE
Stop using proprietary archive compresion methods

### ISSUE THREAD

**Issue by iraizo** [PazerOP/tf2_bot_detector#311]
Since the repo [TF2BD-Community-Lists](https://github.com/ClusterConsultant/TF2BD-Community-Lists) dont have an issue section nor any links to any contact information and since the tf2_bot_detector wiki is linking to the zip file i am talking about i will post this here, the `All.zip` file is using a compression method named [deflate_64](https://en.wikipedia.org/wiki/Deflate#Deflate64/Enhanced_Deflate) this is a version of the `deflate` compression method which is usually used on 99% of the zip files that you want to ship.  
The difference between those is that the `64` version is proprietary, heres a quick rundown of the wikipedia page:
`Deflate64, specified by PKWARE, is a proprietary variant of Deflate. It's fundamentally the same algorithm. What has changed is the increase in dictionary size from 32 KB to 64 KB..` more info bla bla.  
I'd like to request a change of the compression method because why make a project like that open source when using a proprietary compression method that isn't even supposed by every uncompression tool?

**Comment by incontestableness** [comment]
![lift](https://user-images.githubusercontent.com/66852150/113495001-96809c80-94b3-11eb-9608-d789261ce62c.gif)

**Comment by n3oney** [comment]
Bro get off the weed before u create github issues cus i cant comprehend what u just said

**Comment by n3oney** [comment]
.tar.gz plz

**Comment by iraizo** [comment]
> .tar.gz plz

![image](https://user-images.githubusercontent.com/35413558/113495105-44f60300-94ef-11eb-8292-78a4167ef65f.png)

This would even be better compression and supported

**Comment by dotqurter** [comment]
yeah i cant open the deflate method either how do i kick chiter men :(((( 
![money_mouth_face_animated_sticker](https://user-images.githubusercontent.com/29046062/113495204-d51b5480-94bd-11eb-9a33-d9d3ed9f2bed.gif)

**Comment by n3oney** [comment]
i have to agree with that one

**Comment by incontestableness** [comment]
I did check with the zipdetails utility and the archives do in fact use Deflate64. In all seriousness, zip is going to be the most operating system compatible archive format. I don't know what software was used to create the archives, but using a 32 KB dictionary size instead would be simple enough using e.g. 7-zip.

**Comment by iraizo** [comment]
> I did check with the zipdetails utility and the archives do in fact use Deflate64. In all seriousness, zip is going to be the most operating system compatible archive format. I don't know what software was used to create the archives, but using a 32 KB dictionary size instead would be simple enough using e.g. 7-zip.

Correct, but I dont think `.zip` should be used at all when there are better supported compression methods like `.tar.gz`.

**Comment by incontestableness** [comment]
You complained about the usage of the proprietary 64 KB dictionary size implementation. Changing the dictionary size to 32 KB would resolve your issue, as your issue is the proprietary nature of the specification. Zip and deflate are widely supported as archive formats and compression schemes, respectively, on both Windows and Linux, whereas tar is not as supported on Windows and requires additional software to be installed.

> Since the repo TF2BD-Community-Lists dont have an issue section nor any links to any contact information and since the tf2_bot_detector wiki is linking to the zip file i am talking about i will post this here

I've repacked the zips at https://github.com/incontestableness/TF2BD-Community-Lists, but cannot open a pull request as @ClusterConsultant limited that to collaborators. If they want to merge it they can.

**Comment by iraizo** [comment]
> I've repacked the zips at https://github.com/incontestableness/TF2BD-Community-Lists, but cannot open a pull request as @ClusterConsultant limited that to collaborators. If they want to merge it they can.

I appreciate the effort in doing so, I hope this gets fixed asap.

**Comment by ClusterConsultant** [comment]
i'll think about it
